protocol/streaming/full_node_streaming_manager.go

```diff
@@ -410,7 +485,7 @@ func (sm *FullNodeStreamingManagerImpl) SendTakerOrderStatus(
 		clobPairId = takerOrder.OrderId.ClobPairId
 	}
 
-	sm.AddUpdatesToCache(
+	sm.AddOrderbookUpdatesToCache(
```

Comment:
Nit: rename function since this is used for Taker Orders as well

Reply:
renamed to `AddOrderUpdatesToCache`